homeassistant/helpers/llm.py

```diff
@@ -884,3 +894,48 @@ async def async_call(
         ]
 
         return {"success": True, "result": events}
+
+
+class WeatherForecastTool(Tool):
+    """LLM Tool wrapper for weather forecast action."""
+
+    name = f"{WEATHER_DOMAIN}_{SERVICE_GET_FORECASTS}"
+    description = "Get weather forecasts"
+    parameters = vol.Schema(
+        {
+            vol.Required("type"): vol.In(("daily", "hourly", "twice_daily")),
```

Comment:
Why would we include twice daily?

Reply:
To match the service capabilities. I can remove it.
Also while thinking about it I realized that I would also need to add a fallback strategy for entities that do not provide the requested type of forecast.

Reply:
I also wonder if we should follow similar ranges like we do for events. `today` and `week`. Today would return the hourly forecast for next 24h, while week will return for next 7 days daily.